numpy/core/shape_base.py

```diff
@@ -385,8 +387,12 @@ def stack(arrays, axis=0, out=None):
     See Also
     --------
     concatenate : Join a sequence of arrays along an existing axis.
+    vstack : Stack arrays in sequence vertically (row wise).
+    hstack : Stack arrays in sequence horizontally (column wise).
+    dstack : Stack arrays in sequence depth wise (along third axis).
+    column_stack : Stack 1-D arrays as columns into a 2-D array.
```

Comment:
I think omitting these four might have been deliberate, as `stack` is a more predictable replacement for all of these.

Reply:
I'm not following. Do you mean one should always use `stack` instead of these? If so, that should be noted explicitly in the above functions. If not, I don't see why one should not link to these other and more specialized functions.

Reply:
Here's the reference I was looking for: https://github.com/numpy/numpy/issues/7183#issuecomment-180453046

Reply:
#7253 added links in the other direction recommending `stack` be used instead.

Reply:
I've removed the links again as I don't want to go down a long API discussion. This is just intended as a small doc fix.

Note however that the wording was alleviated in #10057. With my user hat on, I still don't know what the recommended way of working is. Can / should I use `vstack` or not? It would be ok to keep it in the API but recommend to use `stack` in which case we shouldn't link from `stack` to `vstack`. As is I find the wording a bit vague. Note also that `concatenate` and `block` still link to the `*stack` variants. That should be removed as well if the intention is to reduce visibility of `*stack`.